website/documentation/content/user_documentation.py

```diff
@@ -105,6 +105,75 @@ def querying():
             ''')
 
 
+doc.text('User Interaction', '''
```

Comment:
Would it be possible to make "User Interaction" a higher-level heading, with two sub-headings:
* Autocomplete
* Selecting options

Reply:
Unfortunately not. The documentation system has no sub-sections.